MAUI/DataGrid/row-drag-and-drop.md

```diff
@@ -0,0 +1,402 @@
+
+To enable row drag and drop functionality, please follow the code example below:
+
+{% tabs %}
+{% highlight XAML %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       ItemsSource="{Binding OrderInfoCollection}"  
+                       AllowDraggingRow="True">
+ </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AllowDraggingRow = true;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="DataGrid row drag and drop" src="Images\row-drag-and-drop\Dragging.gif" width="404"/>
+
+## Dragging scenarios
+
+The SfDataGrid supports drag and drop operations for both data rows and groups in the following scenarios:
+
+ * Records can be reordered to any position using auto-scrolling.
+ * Group positions can be reordered using drag and drop. However, no groups can be added within other groups.
+ * Data rows can be reordered within the same group or moved to other groups as well.
+
+N> Reordering changes are made only in the `SfDataGrid.View`, and not in the underlying data. Thus, the changes will be reverted when performing sorting, grouping, or any other operation refreshing the view. Reordering changes in the underlying data can be achieved by handling a `QueryRowDragging` event in the sample side as explained below in [Reordering underlying data](# Reordering underlying data ).
+
+N> Now you can drag and drop the grid rows using your mouse in the UWP (Desktop) platform.
+
+## Row drag-and-drop template
+
+The data grid allows you to load desired content when performing row drag-and-drop operations using the `SfDataGrid.RowDragDropTemplate`. The template can be defined either in code or XAML.  
+
+## Default template
+
+The default template will be used if no other template is explicitly assigned for row drag-and-drop operations.
+
+<img alt="DataGrid row drag and drop" src="Images\row-drag-and-drop\Dragging.gif" width="404"/>
+
+## Customizing row drag-and-drop template
+
+Any type of custom view can be loaded inside the `SfDataGrid.RowDragDropTemplate`.
+
+Please refer to the following code example that demonstrates how to load a view in a template:
+
+{% tabs %}
+{% highlight xaml %}
+
+        <syncfusion:SfDataGrid.RowDragDropTemplate>
+        <DataTemplate>
+            <Grid BackgroundColor="LightBlue">
+                <Label Text="Drag View"       
+                       TextColor="Black"           
+                       HorizontalTextAlignment="Center" 
+                       VerticalTextAlignment="Center"/>
+            </Grid>
+        </DataTemplate>
+    </syncfusion:SfDataGrid.RowDragDropTemplate>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Default drag and drop" src="Images\row-drag-and-drop\LabelTemp.gif" />
+
+N> Currently, the row drag-and-drop features cannot be used if different row sets with varying heights are specified using the `QueryRowHeight` event.
```

Comment:
Remove the note. This is not applicable.

Reply:
Removed